include.sh

```diff
@@ -5,8 +5,8 @@ BOOTSTRAP_LOGFILE="/var/log/install-cloud-in-a-box.log"
 if [[ -n "$BOOTSTRAP_LOGFILE" ]] ;then
    log_ident="$(basename $0|sed '~s,\.sh,,')"
    echo "Logging bootstrap process to $BOOTSTRAP_LOGFILE for $log_ident"
-   exec 1> >(sed "~s,^,$log_ident | ,"|tee -a $BOOTSTRAP_LOGFILE)
-   exec 2> >(sed "~s,^,$log_ident | ,"|tee -a $BOOTSTRAP_LOGFILE >&2)
+   exec 1> >(sed "~s,^,$log_ident | ," | sudo tee -a $BOOTSTRAP_LOGFILE)
```

Comment:
Why is that needed? When installing CIAB, I have not yet noticed that this is needed. Has anything changed at this point?

Reply:
It's required when using the script on an existing instance. There you normally want to clone the repository as normal user and want to run the script as a normal user.

Reply:
But maybe I only had issues yesterday because I prepare a cloud-in-a-box and started the script again as a normal user?